dspace-spring-rest/src/main/java/org/dspace/app/rest/converter/ItemConverter.java

```diff
@@ -22,11 +28,34 @@
 public class ItemConverter extends DSpaceObjectConverter<org.dspace.content.Item, org.dspace.app.rest.model.ItemRest> {
 	@Autowired
 	private CollectionConverter collectionConverter;
+	@Autowired
+	private BitstreamConverter bitstreamConverter;
 
 	@Override
 	public ItemRest fromModel(org.dspace.content.Item obj) {
 		ItemRest item = super.fromModel(obj);
-		// item.setOwningCollection(collectionConverter.fromModel(obj.getOwningCollection()));
+		item.setInArchive(obj.isArchived());
+		item.setDiscoverable(obj.isDiscoverable());
+		item.setWithdrawn(obj.isWithdrawn());
+		item.setLastModified(obj.getLastModified());
+		try {
+			//item.setTemplateItemOf(collectionConverter.fromModel(obj.getTemplateItemOf()));
+		} catch (Exception e) {
+			e.printStackTrace();
+		}
```

Comment:
BTW NetBeans drops nasty yellow flags on foo.printStackTrace() but approves foo.printStackTrace(System.err) which does the same thing.

However, as this is servlet code, it shouldn't be printing stack traces at all; if needed they should be logged.  [LOG.error("What we think happened", e);]